src/index.js

```diff
@@ -311,6 +314,7 @@ export async function renderVCHTML(data, options = {}) {
     html: templateFn(data),
     orientation,
     templateId,
+    size: 'a4',
```

Comment:
should this come from the template too rather than being hard-coded to a4?

Reply:
all "current templates" are a4 in size